recipes/prometheus-cpp/all/conanfile.py

```
@@ -0,0 +1,112 @@
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        tools.check_min_cppstd(self, 11)
+
+    def requirements(self):
+        if self.options.with_pull:
+            self.requires("civetweb/1.13")
+        if self.options.with_push:
+            self.requires("libcurl/7.74.0")
+        if self.options.with_compression:
+            self.requires("zlib/1.2.11")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["USE_THIRDPARTY_LIBRARIES"] = False
+        self._cmake.definitions["ENABLE_TESTING"] = False
+        self._cmake.definitions["OVERRIDE_CXX_STANDARD_FLAGS"] = False
+
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
+        self._cmake.definitions["ENABLE_PULL"] = self.options.with_pull
+        self._cmake.definitions["ENABLE_PUSH"] = self.options.with_push
+        self._cmake.definitions["ENABLE_COMPRESSION"] = self.options.with_compression
+
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(os.path.join(self._source_subfolder, "LICENSE"), dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "prometheus-cpp"
+        self.cpp_info.names["cmake_find_package_multi"] = "prometheus-cpp"
+        self.cpp_info.names["pkg_config"] = "prometheus-cpp"
```

```suggestion
```

There is no official pkg-config file. This value will be the default for pkg-config generator anyway, but it's misleading to enforce pkg-config name when there is no official pkg-config file.